Add compatibility page (#7821)
opensearch-project/documentation-website: _install-and-configure/install-opensearch/index.md

# Installing OpenSearch

> This section details how to install OpenSearch on your host, including which [ports to open](#network-requirements) and which [important settings](#important-settings) to configure on your host.

Reviewer: ```suggestion
This section provides information about how to install OpenSearch on your host, including which [ports to open](#network-requirements) and which [important settings](#important-settings) to configure on your host.
```

> For operating system compatibility, see [Compatible operating systems]({{site.url}}{{site.baseurl}}/install-and-configure/os-comp/)

Reviewer: ```suggestion
For operating system compatibility, see [Compatible operating systems]({{site.url}}{{site.baseurl}}/install-and-configure/os-comp/).
```

> For operating system compatibility, see [Compatiable operating systems]({{site.url}}{{site.baseurl}}/install-and-configure/os-comp/)

Reviewer 2: ```suggestion
For operating system compatibility, see [Compatible operating systems]({{site.url}}{{site.baseurl}}/install-and-configure/os-comp/)
```
